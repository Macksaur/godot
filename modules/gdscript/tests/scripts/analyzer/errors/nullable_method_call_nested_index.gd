class B:
    func hey():
        pass

class A:
    var b: B?

func test():
    var foo: Array[A?] = [A.new()]
    foo[0].hey()