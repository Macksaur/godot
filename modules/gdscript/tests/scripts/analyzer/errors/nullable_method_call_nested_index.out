GDTEST_ANALYZER_ERROR
Potentially unsafe for nullable "foo[0]" to call "hey()".
